### PR TITLE
Pass all scopes in token request and update IndividualLocalAuth template

### PR DESCRIPTION
**Pull request by captainsafia**
Addresses https://github.com/dotnet/aspnetcore/issues/25961, https://github.com/dotnet/aspnetcore/issues/26218

### Description

This PR updates the code where authentication scopes are passed to the MSAL API to include all scopes provided by the user:

- Include scopes defined in `extraScopesToConsent` in sign-in request
- Include scopes requested in access token, default scopes, and extra scopes when getting the user
   - This was handled in MSALv1 by the `getAccount()` method but this logic has been removed in MSALv2
- Updates template string in IndividualLocalAuth template

### Customer Impact

Without this fix, end-users will be unable to authenticate into applications configured with Azure AD if they've been configured with additional scopes.

### Regression

This is a regression from Blazor WASM 3.2 as a result of the upgrades to MSAL v2.

### Risk

Low, this change is scoped to AzureAD authentication scenarios in Blazor WASM.

### CODE REVIEW

**Comment by mkArtakMSFT** [comment]
Approved by Tactics for RC2.

File: src/Components/WebAssembly/Authentication.Msal/src/Interop/AuthenticationService.ts [hunk 1]
```diff
@@ -81,10 +82,23 @@ class MsalAuthorizeService implements AuthorizeService {
             return;
         }
 
+        const scopes: string[] = [];
+        if (this._settings.defaultAccessTokenScopes && this._settings.defaultAccessTokenScopes.length > 0) {
+            scopes.concat(this._settings.defaultAccessTokenScopes)
```
Comment (pranavkm) [inline src/Components/WebAssembly/Authentication.Msal/src/Interop/AuthenticationService.ts:87]: Doesn't concat return a new array?

**Review by pranavkm — COMMENTED** [review]
Something's broken in the build

**Review by pranavkm — APPROVED** [review]
I'm guessing these are all manually tested.

File: src/Components/WebAssembly/Authentication.Msal/src/Interop/AuthenticationService.ts (hunk 1 above)
Comment (captainsafia) [reply src/Components/WebAssembly/Authentication.Msal/src/Interop/AuthenticationService.ts]: 😲  Fixed.

**Comment by captainsafia** [comment]
> I'm guessing these are all manually tested.

Tested via the samples at https://github.com/captainsafia/blazor-auth-samples.

**Pull request merged**